src/main/java/com/eunu/springbootboard/controller/PostController.java

```diff
@@ -0,0 +1,46 @@
+package com.eunu.springbootboard.controller;
+
+import com.eunu.springbootboard.ApiResponse;
+import com.eunu.springbootboard.dao.post.PostDto;
+import com.eunu.springbootboard.dao.post.PostService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PostController {
+    @Autowired
+    PostService postService;
+
+    @GetMapping("/posts/{postId}")
+    public ApiResponse<PostDto> getOne(@PathVariable Long postId) throws NotFoundException {
+        PostDto one = postService.findOne(postId);
+        return ApiResponse.ok(one);
+    }
```

Comment:
```suggestion
    @GetMapping("/posts/{postId}")
    public ApiResponse<PostDto> getOne(@PathVariable Long postId) {
        PostDto one = postService.findOne(postId);
        return ApiResponse.ok(one);
    }
```

NotFoundException는 unchecked exception 아닌가요? 작성할 필요가 없어보입니다.

또 one이라는 네이밍은 개인적으로 이해하기 어렵네요. 누구나 이해할 수 있는 변수명을 사용하면 좋겠습니다!